cinema/views.py

```diff
@@ -127,6 +122,29 @@
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="actors",
+                type={"type": "array", "items": {"type": "number"}},
+                description="Filter by actor id (ex. ?actors=2,3)",
+            ),
+            OpenApiParameter(
+                name="genres",
+                type={"type": "array", "items": {"type": "number"}},
+                description="Filter by genre id (ex. ?genres=2,3)",
+            ),
+            OpenApiParameter(
+                name="title",
+                type={"type": "string"},
+                description="Filtering by part of the title or full title "
+                            "(ex. ?title=Inc)",
+            )
+        ]
+    )
+    def list(self, request, *args, **kwargs):
+        return super().list(request, *args, **kwargs)
+
```

Comment:
The `extend_schema` decorator is used to enhance the schema for the `list` action in `MovieViewSet`. This is a good practice for providing detailed API documentation. Ensure that the parameter descriptions are clear and accurate.